File: plugins/approvej-intellij-plugin/src/unitTest/java/org/approvej/intellij/CompareReceivedWithApprovedActionTest.java
package org.approvej.intellij;

import static org.assertj.core.api.Assertions.assertThat;

import com.intellij.openapi.actionSystem.ActionUpdateThread;
import org.junit.jupiter.api.Test;

class CompareReceivedWithApprovedActionTest {

  @Test
  void getActionUpdateThread() {
    assertThat(new CompareReceivedWithApprovedAction().getActionUpdateThread())
        .isEqualTo(ActionUpdateThread.BGT);
  }
}

File: plugins/approvej-intellij-plugin/src/unitTest/kotlin/org/approvej/intellij/CompareReceivedWithApprovedActionTest.kt
package org.approvej.intellij

import com.intellij.openapi.actionSystem.ActionUpdateThread
import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class CompareReceivedWithApprovedActionTest {

  @Test
  fun getActionUpdateThread() {
    assertThat(CompareReceivedWithApprovedAction().actionUpdateThread)
      .isEqualTo(ActionUpdateThread.BGT)
  }
}
